.github/workflows/contributor.yml

```diff
@@ -8,12 +8,7 @@ on:
 jobs:
   test:
     if: >
-      github.event.pull_request.head.repo.fork == true ||
-      !(
```

Comment:
I was trying to see if there is a setting that allows to control group membership, but it does not appear that this is the case i.e. [check permission](https://michaelheap.com/github-actions-check-permission/).